File: app/src/main/java/com/iuicity/xinjr/feature/one/OneFragment.java
package com.iuicity.xinjr.feature.one;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.iuicity.xinjr.R;
import com.iuicity.xinjr.base.BaseFragment;

/**
 * Created by Shui on 2017/7/6.
 */

public class OneFragment extends BaseFragment {
    public static OneFragment newInstance() {

        Bundle args = new Bundle();

        OneFragment fragment = new OneFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View inflate = inflater.inflate(R.layout.fragment_one, container, false);
        return inflate;
    }
}

File: app/src/main/java/com/iuicity/xinjr/feature/one/OneFragment.kt
package com.iuicity.xinjr.feature.one

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.iuicity.xinjr.R
import com.iuicity.xinjr.base.BaseFragment

/**
 * Created by Shui on 2017/7/6.
 */

class OneFragment : BaseFragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val inflate = inflater!!.inflate(R.layout.fragment_one, container, false)
        return inflate
    }

    companion object {
        fun newInstance(): OneFragment {

            val args = Bundle()

            val fragment = OneFragment()
            fragment.arguments = args
            return fragment
        }
    }
}
